### PR TITLE
Add initial MAINTAINERS file

### PR DESCRIPTION
**Pull request by YHLiang85**
The initial maintainers are Will Liang and George Hung.

Signed-off-by: Will Liang <will.liang@quantatw.com>

### CODE REVIEW

**Comment by bjwyman** [comment]
The OpenBMC project does not use pull requests (for most repos). Changes such as this should be submitted through Gerrit.
https://github.com/openbmc/docs/blob/master/development/gerrit-setup.md

**Comment by YHLiang85** [comment]
> The OpenBMC project does not use pull requests (for most repos). Changes such as this should be submitted through Gerrit.
> https://github.com/openbmc/docs/blob/master/development/gerrit-setup.md

Because the gerrit/github has synchronization problem, Brad told me use github pull requests until they gerrit/github synchronization problem can be solved.

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    comment); it is important to follow the correct syntax and semantics as we
+    may want to use automated tools with this file in the future.
+
+    A change cannot be added to an OpenBMC repository without a MAINTAINER's
+    approval; thus, a MAINTAINER should always be listed as a reviewer.
+
+START OF MAINTAINERS LIST
+-------------------------
+
+M:  Will Liang <will.liang@quantatw.com> <willliang!>
```
Comment (BenjaminFair) [inline MAINTAINERS]: Make sure to register these IRC nicks

Comment (YHLiang85) [reply MAINTAINERS]: willliang and ghung accounts are registered at https://irc-source.com/

**Comment by BenjaminFair** [comment]
You'll want to register it with Freenode's NickServ. More info:
https://freenode.net/kb/answer/registration

**Comment by YHLiang85** [comment]
Both  IRC nicks have registered on freenode.

**Pull request closed without merging**